modules/util/test/spec.coffee

```diff
@@ -303,6 +303,23 @@ describe "Util", ->
     hx.isPlainObject(null).should.equal(false)
     hx.isPlainObject(undefined).should.equal(false)
 
+  it 'isNumber: should work', ->
+    class A
+      constructor: ->
+
+    hx.isNumber(new A).should.equal(false)
```

Comment:
Could also do `hx.isNumber(A)`? Although I guess this is probably equivalent to `hx.isNumber(->)`